### PR TITLE
feat : added styles for texts

### PR DESCRIPTION
**Pull request by aatishpandey**
image
Includes:
-headings
-small text
-gray text
-center text
-left text
-right text

### CODE REVIEW

File: stylesheets/typography.css [hunk 1]
```diff
@@ -6,3 +6,55 @@
...
+}
+
+.h6{
+    font-size:1.25rem;
+}
+
+/* para */
+.para{
+    font-weight: 400;
+}
```
Comment (shuntzhere) [inline stylesheets/typography.css]: You can give this a more generic name like font-md

**Review by shuntzhere — COMMENTED** [review]
Looks good, suggested a minor change

**Review by shubhamkumar648 — APPROVED** [review]
as i seeclean is code suggestion :- you can also more generic name p-m, p-l, remaining good

File: stylesheets/typography.css (hunk 1 above)
Comment (aatishpandey) [reply stylesheets/typography.css]: thanks for pointing out , fixed in recent commit

**Pull request merged**